docs/pages/database-access/faq.mdx

```diff
@@ -3,76 +3,102 @@ title: Database Access FAQ
 description: Frequently asked questions about Teleport Database Access.
 ---
 
-# Database Access FAQ
-
 ## Which database protocols does Teleport Database Access support?
 
-Teleport Database Access currently supports PostgreSQL, MySQL, MariaDB and MongoDB
-protocols.
+Teleport Database Access currently supports the PostgreSQL, MySQL, MariaDB and
+MongoDB protocols.
```

Comment:
Since you're here anyway, can you add Microsoft SQL Server and Redis to the list?

Reply:
Added. I've also added Azure Database to the list of Cloud-hosted databases we support.